### ISSUE TITLE
Human intervention into the training process ???

### ISSUE THREAD

**Issue by KeepOnCarvingHuang** [gzrjzcx/ML-agents#1]
I have always had a question about mlagents: they randomly select actions at the beginning of training. Can we incorporate human intervention into the training process of mlagents to make them train faster? Is there a corresponding method in mlagents? Looking forward to your answer.

**Comment by gzrjzcx** [comment]
Of course you can. In fact, there are many ways to do that. We prefer pre-trained model (like supervised learning) rather than initial random model to start RL training in practice, it is also as know as fine-tuning.  In addition, we also use human knowledge to shape reward, or create curriculum for RL learning, it's also viewed as human prior signal for RL training. Finally, there are also some different RL paradigms like Inverse RL to utilize human knowledge.
